src/components/AssetTree/AssetTree.tsx

```diff
@@ -76,9 +76,10 @@ class AssetTree extends Component<AssetTreeType> {
     if (treeNode.props.children) {
       return;
     }
-    const assetId = treeNode.props.eventKey;
+    const eventKey = treeNode.props.eventKey;
+    const assetId = eventKey ? Number.parseInt(eventKey, 10) : undefined;
```

Comment:
```suggestion
    const assetId = eventKey ? Number.parseInt(eventKey) : undefined;
```

Reply:
Linting fails when removing this. It is actually recommended to keep this param, otherwise parseInt will try to determine the radix itself based on the input string.